nb.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
Did you mean to add this file to your PR? If not, definitely be wary of what you're adding to your commits!